### DISCUSSION TITLE
How are servers from SERVER_NAMES chosen?

### DISCUSSION THREAD

**Discussion by sunbeam60** [qdm12/gluetun#924] [Q&A] [answered]
I had imagined the list of servers were attempted in order, with follow-on servers being fall-back if preceeding servers failed.

But it seems the server to dial is chosen at random or, perhaps, as a rotation.

I couldn't find anything in the wiki that discussed this.

**Comment by qdm12** [comment, marked as answer]
Yes it's random for now from the servers filtered out. I'll eventually get to this and make it remember what servers failed etc. but it's not a priority really for now.

**Reply by sunbeam60** [reply]
All good - I was looking for clarity more than changes. Thanks for getting back and for gluetun and deunhealth.
